Chart: update sortingMethod description (#4977)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChart/1 Configuration/dataPrepareSettings/sortingMethod.md

[note]We recommend setting this property to **false** if objects in the data source are already sorted properly. This will improve UI component performance.

> [note]When the [argumentType](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/argumentAxis/argumentType.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#argumentType') is *'numeric'* or *'datetime'*, the **sortingMethod** property does not affect argument axis. If you want to display arguments in descending order, enable the [inverted](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#inverted) property.

Reviewer: ```suggestion
[note]When the [argumentType](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/argumentAxis/argumentType.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#argumentType') is *'numeric'* or *'datetime'*, the **sortingMethod** property does not affect the argument axis. If you want to display arguments in descending order, enable the [inverted](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#inverted) property.
```

> Alternatively, you can specify the drawing order using a comparison function. The function accepts two data source objects and should return the value on which the order will depend. For example, assume that *A* and *B* objects represent two series points. If the function returns a value less than 0, point *A* will be drawn _before_ point *B*. If the function returns a value greater than 0, point *A* will be drawn _after_ point *B*. If the function returns 0, the drawing order of *A* and *B* remains unchanged.

Reviewer: ```suggestion
Alternatively, you can use a comparison function to specify the drawing order. The function accepts two data source objects and should return the value on which the order depends. For example, *A* and *B* objects represent two series points. If the function returns a value less than 0, point *A* is drawn _before_ point *B*. If the function returns a value greater than 0, point *A* is drawn _after_ point *B*. If the function returns 0, the drawing order of *A* and *B* remains unchanged.
```

> [note]When the [argumentType](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/argumentAxis/argumentType.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#argumentType') is *'string'*, the **sortingMethod** property does not accept Boolean values. However, you can still apply sorting using a comparison function. Alternatively, use the [categories](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/argumentAxis/categories.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#categories') array to perform the same task. Note that the **categories** array takes precedence over the **sortingMethod** function.

Reviewer: ```suggestion
[note]When the [argumentType](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/argumentAxis/argumentType.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#argumentType') is *'string'*, the **sortingMethod** property does not accept Boolean values. However, you can still apply sorting using a comparison function. Alternatively, use the [categories](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/argumentAxis/categories.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#categories') array to perform the same task. Note that the **categories** array has priority over the **sortingMethod** function.
```

> The following code sorts string arguments in the alphabetical order:

Reviewer: ```suggestion
The following code sorts string arguments in alphabetical order:
```
